steal-qunit.js

```diff
@@ -111,8 +111,8 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
+
+		if(QUnit.assert && QUnit.assert.ok && QUnit.assert.async) {
```

Comment:
QUnit.load() works in all 1.x versions. I would just keep it simple and do the check for 2 as the major version.